### DISCUSSION TITLE
Describe bot level definitions

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#4] [Q&A] [answered]
I want to know what the bot level of beginner to master means.
Are higher levels "faster", uses a wider range of words in a list by frequency of use, or tries more words (if using, say a depth or breadth in search across the word list)

**Comment by azadbolour** [comment, marked as answer]
This is described in [the EagerWords FAQ](https://eagerwords.com/faq.html). 

"
Higher levels have larger vocabularies at their disposal. The master bot level uses the entire EagerWords dictionary of about 76000 words.
"
Other than that, at this time, the search algorithm for optimal matches is the same.

Because the exact boundaries between different levels are in the process of being fine-tuned, they are not explicitly specified in the documentation for all levels at this time. The basic bot currently knows 4000 words.

**Reply by danconde** [reply]
Got it.  I was looking at the docs on the mobile app which did not list it, but it links to the FAQ at the "Playing with Other Users" page where I could have found it.
